src/main/java/mate/academy/spring/validation/Email.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Target({ElementType.PARAMETER, ElementType.FIELD})
```

Comment:
We should either remove the parameter target. or add `@Email` to the `get user by email` parameter to validate it too